doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3084,6 +3084,12 @@ the conversion in hypotheses :n:`{+ @ident}`.
    compilation cost is higher, so it is worth using only for intensive
    computations.
 
+   .. flag:: NativeCompute Timing
+
+      This option causes all calls to the native compiler to print
```

Comment:
a flag (boolean), not an option (non-boolean value)

Reply:
They seem to be called "option"s in a bunch of other places, though.  e.g., `NativeCompute Profiling`, `Debug Cbv`, `Extraction Optimize`, `Extraction Conservative Types`, `Extraction SafeImplicits`, `Printing Coercions`, `Simplex`, `Lia Cache`, `Nia Cache`, `Nra Cache`, `Stable Omega`, `Omega UseLocalDefs`, `Omega System`, `Omega Action`, `Program Cases`, `Program Generalized Coercion`, `Shrink Obligations`, `Universe Polymorphism`, `Polymorphic Inductive Cumulativity`, `Strict Universe Declaration`, `Private Polymorphic Universes`, (need I go on?  this is only maybe half of the entries listed under `.. flag` which call the flag an option).  I think, for consistency, if you want them to be called `flag`s in the manual, you should change them all at once.  Since both flags close to this one use "option" to describe them, I think it's probably better in this PR to call this an "option".

Reply:
We've been correcting these as we run across them.  It's incongruent to see "Flag" in the box and "option" immediately below it.  And of course the syntax for setting/clearing a flag and for setting an option is different as described here: https://coq.gitlab.io/-/coq/-/jobs/339082210/artifacts/_install_ci/share/doc/coq/sphinx/html/proof-engine/vernacular-commands.html#flags-options-and-tables

![image](https://user-images.githubusercontent.com/1253341/68045045-e0010280-fc95-11e9-80f8-0be7608a8adc.png)

Reply:
> if you want them to be called flags in the manual, you should change them all at once

See https://github.com/coq/coq/pull/11050.

Reply:
This has been fixed elsewhere, so should be fixed here as well:

```suggestion
      This flag causes all calls to the native compiler to print
```

Reply:
Fixed